docs/enterprise/sso.md

```diff
@@ -0,0 +1,402 @@
+
+  auth:
+    cookie_auth_verification:
+      auth_source: saml
+      friendly_name_claim_name: displayName
+      external_user_id_claim_name: samName
+      default_initial_groups:
+        - "Developers"
+    logout_redirect_url: "https://lakefs.company.com/logout-saml"
+    encrypt:
+      secret_key: shared-secrey-key
+    ui_config:
+      login_url: "https://lakefs.company.com/sso/login-saml"
+      logout_url: "https://lakefs.company.com/sso/logout-saml"
+      login_cookie_names:
+        - internal_auth_session
+        - saml_auth_session
+      rbac: external
+```
+
+Fluffy Configuration:
+```yaml
+fluffyConfig: &fluffyConfig |
+  logging:
+    format: "json"
+    level: "INFO"
+    audit_log_level: "INFO"
+    output: "="
+  installation:
+    fixed_id: fluffy-authenticator
+  auth:  
+    # better set from secret FLUFFY_AUTH_ENCRYPT_SECRET_KEY must be equal to what is used in lakeFS for auth_encrypt_secret_key
+    encrypt:
+      secret_key: shared-secrey-key    
+    logout_redirect_url: https://lakefs.company.com
+    post_login_redirect_url: https://lakefs.company.com
+    saml:
+      enabled: true 
+      sp_root_url: https://lakefs.company.com
+      sp_x509_key_path: '/etc/saml_certs/rsa_saml_private.cert'
+      sp_x509_cert_path: '/etc/saml_certs/rsa_saml_public.pem'
+      sp_sign_request: true
+      sp_signature_method: "http://www.w3.org/2001/04/xmldsig-more#rsa-sha256"
+      idp_metadata_url: "https://adfs-auth.company.com/federationmetadata/2007-06/federationmetadata.xml"
+      # idp_authn_name_id_format: "urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified"
+      external_user_id_claim_name: samName
+      # idp_metadata_file_path: 
+      # idp_skip_verify_tls_cert: true
+```
+
+### OpenID Connect
+{: .d-inline-block }
+Enterprise
+{: .label .label-purple }
+
+In order for fluffy to work some values must be configured, update the relevant values section in this file and comment it out. (lakefsConfig and fluffyConfig values)
+1. Replace `lakefsConfig.friendly_name_claim_name` and `fluffyConfig.friendly_name_claim_name` with the right claim name
+2. Replace `fluffyConfig.auth.logout_redirect_url`:  https://oidc-provider-url.com/logout/path with full URL to logout API of the OIDC provide
+3. Replace `fluffyConfig.auth.oidc.url` with full url to OIDC provider  https://oidc-provider-url.com
+4. Replace in `fluffyConfig.auth.oidc.logout_endpoint_query_parameters` with parameters for the target OIDC provider for logout. In this example it's API auth0 returnTo=https://lakefs.company.com/oidc/login
+5. Replace the values in `fluffyConfig.auth.oidc.client_id` and `fluffyConfig.auth.oidc.client_secret` for OIDC. 
+6. Replace the values in `logout_client_id_query_parameter`, note it should match the the key/query param that represents the client id and required by the specific OIDC provider
+7. Replace `lakefs.company.com` with the lakeFS server endpoint.
+
+lakeFS Enterprise Configuration:
```

Comment:
```suggestion
lakeFS Server Configuration:
```

Reply:
can we tell the user where to find this file?

Reply:
replied above